ActoKids/web-crawler
# Mike feature/add package and dynamo fix

Okay gang here we are. This is the final pull request unless anyone has anything else to make a comment on. 

Let me know how you guys are feeling about this.
To test -
Run the scripts on lambda.
Check this link : https://api.2edusite.com/v1/events - Should be populated with crawler events.
Check this link : http://www-dev.2edusite.com/dashboard - Should be populated with crawler events.

Thanks to @daonguyen81 and @rberry206  for help on the crawlers. (@daonguyen81  helped SS and OFA, @rberry206  for providing google crawler)

If data already exists then ask devop's to delete.
Thanks :)
